### DISCUSSION TITLE
Synchronisation: What's this recurring timer with no date or interval blocks test?

### DISCUSSION THREAD

**Discussion by jzaefferer** [wix/Detox#4161] [Q&A]
Every now and then a test in CI (Bitrise) ends up blocked by a recurring timer like this:
```
• Run loop "Main Run Loop" is awake.
• 1 enqueued native timers:
  - Timer #1:
    + Fire date: none.
    + Time until fire: 0.000.
    + Repeat interval: 0.
    + Is recurring: YES.
• There are 1 work items pending on the dispatch queue: "Main Queue (<OS_dispatch_queue_main: com.apple.main-thread>)".
```
It says it's recurring, but it has no fire date, no time until fire and no repeat interval, so it's giving me zero clue where it could come from, or even how to fix it.

When it happens, it happens many times at once, like 14 or more.

I've never noticed any pattern around this - it only happens sometimes (<5% of tests) and affects different tests.

Any hint what could be causing this kind of timer?

**Comment by jzaefferer** [comment]
Current hypothesis: This is could be related to https://github.com/software-mansion/react-native-reanimated/issues/3879 - maybe upgrading from 2.14.4 to 2.15.0 will help.

**Reply by jzaefferer** [reply]
I did not help :/

**Comment by micahdas** [comment]
Hi @jzaefferer Did you ever figure out how to resolve this?

**Reply by jzaefferer** [reply]
Nope, unfortunately I didn't.
